### PR TITLE
RAD-39 Add Optical_Element to Reference File Schemas

### PR DESCRIPTION
**Pull request by PaulHuwe**
Added optical_element to appropriate reference file schemas. 
Added ma_table_name to dark schema. 
Adjusted pixelarea schema imports.

### CODE REVIEW

**Comment by PaulHuwe** [comment]
I also updated the optical_element test.

File: src/rad/resources/schemas/reference_files/ref_optical_element-1.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+type: object
+properties:
+  instrument:
+    type: object
+    properties:
+      optical_element:
+        allOf:
+          - $ref: ../wfi_optical_element-1.0.0
+          - title: Name of the filter element used
```
Comment (gardner358) [inline src/rad/resources/schemas/reference_files/ref_optical_element-1.0.0.yaml:16]: Doesn't wfi_optical_element-1.0.0 already have a title?

**Review by ddavis-stsci — APPROVED** [review]
Looks Good.

File: src/rad/resources/schemas/reference_files/ref_optical_element-1.0.0.yaml (hunk 1 above)
Comment (PaulHuwe) [reply src/rad/resources/schemas/reference_files/ref_optical_element-1.0.0.yaml:16]: It does.. I put it here as potential for overriding (should we decide to change anything for the ref version), but I guess it is superfluous and can be removed.

Comment (PaulHuwe) [reply src/rad/resources/schemas/reference_files/ref_optical_element-1.0.0.yaml:16]: Removed.

**Pull request merged**